tests/TestHelpers/GraphHelper.php

```diff
@@ -490,18 +489,17 @@ public static function addUsersToGroup(
 		string $adminUser,
 		string $adminPassword,
 		string $groupId,
-		array $users
+		array $userIds
 	): ResponseInterface {
-		$url = self::getFullUrl($baseUrl, 'groups/' . $groupId . '/users');
-		$payload = [
-			"members@odata.bind" => []
-		];
-		foreach ($users as $user) {
-			$payload[0][] = self::getFullUrl($baseUrl, 'users/' . $user["id"]);
+		$url = self::getFullUrl($baseUrl, 'groups/' . $groupId);
+		$payload = [ "members@odata.bind" => [] ];
+		foreach ($userIds as $userId) {
+			$payload["members@odata.bind"][] = self::getFullUrl($baseUrl, 'users/' . $userId);
 		}
-		return HttpRequestHelper::post(
+		return HttpRequestHelper::sendRequest(
```

Comment:
I'm surprised that it worked before with a POST request

Reply:
this method was unused before